builds/azure-pipelines/build-release.yml

```diff
@@ -0,0 +1,40 @@
+trigger: none
+
+pr: none
+
+schedules:
+  - cron: "0 0 * * Mon-Fri"
```

Comment:
consider doing Sun-Thurs since it's running at night & we will likely look at it the next day

Reply:
Changed to run everyday, pipeline runs in under 5 minutes. We can change this to a lower frequency if it becomes too taxing in the future.